### PR TITLE
Feat : Added Cryptocurrency calculator as DAY-75 Project

### LINKED ISSUE #281: feat : Want to add a Cryptocurrency Calci as a project
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
This Cryptocurrency Calculator helps users convert the price of various cryptocurrencies to different fiat currencies. By fetching real-time data from an API, it ensures that you always get the most up-to-date prices for your calculations.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Meetjain1**
## Related Issue

[Cite any related issue(s) this pull request addresses. If none, simply state "None”]
write issue no. here
Closes #281 ,
Added README.md file also with assets , animation to provide Best user friendly experience.

## Email id used to regsiter for VSoc'24
meetofficialhere@gmail.com

## Description

[Please include a brief description of the changes or features added]
### PR Description

This Cryptocurrency Calculator helps users convert the price of various cryptocurrencies to different fiat currencies. By fetching real-time data from an API, it ensures that you always get the most up-to-date prices for your calculations.

## Tech Stacks:

- HTML
- CSS
- JavaScript

## How It Works

1. **User Input**: Enter the amount of cryptocurrency you wish to convert.
2. **Select Currency**: Choose the cryptocurrency and the fiat currency you want to convert to.
3. **Fetch Real-Time Data**: The calculator fetches the latest price data from a reliable API.
4. **Calculate and Display**: The calculator computes the converted amount and displays the result in an easy-to-read format with attractive fonts and colors.

## Features

- **Real-Time Data**: Always get the latest prices with API integration.
- **Responsive Design**: Works seamlessly on mobile, tablet, and laptop screens.
- **Interactive UI**: Includes hover effects and animations for a smooth user experience.
- **Attractive Background**: A visually appealing background image enhances the user interface.

Using this calculator users can stay updated with the current cryptocurrency market and make informed decisions for the investments.

## Type of PR

- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Security enhancement
- [ ] Other (specify): _______________

check in issue by entering [X] in boxes

## Screenshots / Videos (if applicable)

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [X] I have performed a self-review of my code.
- [X] I have read and followed the Contribution Guidelines.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have followed the code style guidelines of this project.
- [X] I have checked for any existing open issues that my pull request may address.
- [X] I have ensured that my changes do not break any existing functionality.
- [ ] Each contributor is allowed to create a maximum of 4 issues per day. This helps us manage and address issues efficiently.
- [X] I have read the resources for guidance listed below.
- [X] I have followed security best practices in my code changes.

check in issue by entering [X] in boxes
## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Contribution Guidelines

Thank you for considering contributing to our project! To ensure smooth collaboration and effective contribution management, please adhere to the following guidelines:

### Issue Creation

1. **Limit on Issues:**
   - Each contributor is allowed to create a maximum of **4 issues per day**. This helps us manage and address issues efficiently.

### Contribution Levels

2. **Basic Contributions:**
   - This project is primarily focused on documentation. Most of the setup has been completed, so contributors will generally need to work on basic code tasks, such as writing tests.
   - For these tasks, issues will be assigned the **Easy** label.

3. **Acknowledging Hard Work:**
   - If a contributor puts in significant effort on a task, the issue will be upgraded to **Medium**. This is our way of recognizing and appreciating extra effort.

4. **Feature Additions and Component Work:**
   - Contributors working on new features or components using JSX/TSX will be assigned a level based on the complexity and quality of their work.
   - The more complex and valuable the contribution, the higher the level assigned.

### Level Definitions

- **Easy:**
  - Tasks are straightforward, such as fixing minor bugs, writing tests, or making simple documentation updates.
- **Medium:**
  - Tasks require more effort, such as addressing complex bugs, improving existing features, or making substantial documentation improvements.
- **Hard:**
  - Tasks are highly complex and involve significant new feature development, major refactoring, or extensive contributions to the project’s core components.

We look forward to your contributions and appreciate your effort in helping us improve the project!

### CODE REVIEW

**Comment by Meetjain1** [comment]
@dhairyagothi have a look on this pr . Thanks

**Comment by dhairyagothi** [comment]
@Meetjain1 please solve conflicts 
- merge main branch first

File: public/index.html [hunk 1]
```diff
@@ -458,11 +458,17 @@
...
+            <div class="project-name">Stock Profit Calculator</div>
             <a href="Stock-Profit-Calculator/index.html" class="demo-link">Demo</a>
-      </div>
+        </div>
+
+        <div class="table">
+            <div class="day-number">Day 75</div>
+            <div class="project-name">Crypto Currency Calculator</div>
+            <a href="Cryptocurrency-Calculator/index.html" class="demo-link">Demo</a>
+        </div>
```
Comment (dhairyagothi) [inline public/index.html]: here is conflict 
- merge main branch

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Comment by Meetjain1** [comment]
@dhairyagothi resolved can you please check .

**Review by dhairyagothi — APPROVED** [review]

**Comment by Meetjain1** [comment]
@dhairyagothi please add points also in the leaderboard . Thanks

**Pull request merged**